vendor/blockchain_contracts/tests/rfc003_bitcoin_htlc.rs

```diff
@@ -60,14 +65,17 @@ fn fund_htlc(
         PrivateKey::from_str("cNZUJxVXghSri4dUaNW8ES3KiFyDoWVffLYDz7KMcHmKhLdFyZPx").unwrap();
     let refund_keypair: KeyPair = refund_privkey.key.clone().into();
     let refund_pubkey_hash: PubkeyHash = refund_keypair.public_key().clone().into();
-    let refund_timestamp = Timestamp::now().plus(10);
+
+    let current_time = client.get_blockchain_info().unwrap().mediantime;
+
+    let refund_timestamp = Timestamp::from(current_time as u32).plus(10);
```

Comment:
❓Does this mean we will always need to wait at least 10 seconds before we can refund? Can we reduce this to speed up the tests?
```suggestion
    let refund_timestamp = Timestamp::from(current_time as u32).plus(2);
```

Reply:
We can reduce this yeah. 2 is probably a bit low because I am asserting that the HTLC cannot be refunded before the refund timestamp, hence setting this to low might result in a flacky test.

I'll set it to 5 and test if it is stable :)